tests/005-test-volume_mount.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+job_name=volume_mount
+
+# test volume_mount
+test_volume_mount_nomad_job() {
+    pushd ~/go/src/github.com/Roblox/nomad-driver-containerd/example
+
+    setup_bind_source
+
+    echo "INFO: Starting nomad $job_name job using nomad-driver-containerd."
+    nomad job run $job_name.nomad
+
+    echo "INFO: Checking status of $job_name job."
+    local job_running=1
+    for i in 1 2 3 ; do
+        job_status=$(nomad job status -short $job_name|grep Status|awk '{split($0,a,"="); print a[2]}'|tr -d ' ')
+        if [ "$job_status" != "running" ];then
+            echo "Info: Job status of ${job_name} is $job_status, wait 2 second."
+            continue
+        else
+            job_running=0
+            break
+        fi
+    done
```

Comment:
Why do we need this `for` loop (`for i in 1 2 3`) when we are waiting for the container to get into running state at `L35: is_${job_name}_container_active`?

Reply:
The job_status here is `pending` and then `running` when i test locally. 

May be we can move the checking after `is_${job_name}_container_active` checking to avoid the loop?

Reply:
Yes, we should move the `job_status` check after `is_${job_name}_container_active`

We can also remove the inspect check #L37-L42.